WebHost.py

```diff
@@ -36,13 +37,14 @@ def get_app():
     return app
 
 
-def create_ordered_tutorials_file():
+def create_ordered_tutorials_file() -> typing.List[typing.Dict[str, typing.Any]]:
     import json
-    with open(os.path.join("WebHostLib", "static", "assets", "tutorial", "tutorials.json")) as source:
+    with open(Utils.local_path("WebHostLib", "static", "assets", "tutorial", "tutorials.json")) as source:
         data = json.load(source)
     data = sorted(data, key=lambda entry: entry["gameTitle"].lower())
-    with open(os.path.join("WebHostLib", "static", "generated", "tutorials.json"), "w") as target:
+    with open(Utils.local_path("WebHostLib", "static", "generated", "tutorials.json"), "w") as target:
```

Comment:
I have not changed WebHost to use `*_path` back when I implemented `user_path` because this would require a split in static between local_path (read-only) and user_path (generated), but I think using `local_path` everywhere as a stop-gap is fine :+1: - should still think about how to fix it for read-only install dir one day, which could also be considered a security measure (i.e. the program can not change the program code on disk once we do that).